### DISCUSSION TITLE
Gem Dealer trades

### DISCUSSION THREAD

**Discussion by BarchamMal** [BarchamMal/MC-Extended#23] [Polls]
I've heard a word or two or one hundred and seventeen, about gem dealer trades needing changed. is this the case? please pole your pole.

**Comment by Raven-dreams** [comment]
Again, Fortune is more commonly used than Silk Touch.  Silk Touch is more useful for getting Ice or other items that break on mining. 
Fortune is more likely to be carried around when looking for resources, and if you don't want to carry extra tools, then people will often carry a Silk Touch and bring it back and then use Fortune.   

The current system is a sunk fallacy, if your trying to just use it to level a Villager trade options.  Your loosing more than you can potentially gain.  

Also you can't just use the resource in question to reform the ore block, you get the building block instead, so that method isn't viable either.

As for realism, this is a fantasy computer game where blocks float in the air and need no support, I think the idea of realism can be ignored.

Something that could be a better option would be to use a resource that still makes sense, but a player would either have an abundance of or give them a reason to want to mine it more often, (i.e. copper in its current state).  

A trade of 2 Emeralds and 1 ore block of Diamond for just 2 Diamond isn't a good value trade.  An example of better value could be say 2 Emeralds and X number of copper ingots/blocks for 2 Diamonds.  Of course a larger number would be a better value of common item to rare item conversion.  But that number would have to be tested and evaluated each time its applied to a trade option.

**Reply by BarchamMal** [reply]
Okay, that raises the number much higher than one hundred and seventeen. I agree with you; I never thought you were wrong. However, I don’t want to make too many significant changes to the mod at once, as it can be quite frustrating. Nevertheless, I think I’ll follow your suggestion. Not today or right now, but sometime in the future. (I assume you’re not a coder, so I won’t ask you to implement this yourself.) You could create an issue with all or several of the changes you suggest. I certainly _will_ do this soon.

**Reply by Raven-dreams** [reply]
I was going to try and do it myself and test it out.  I just couldn't find the information I was looking for.  The last time I was in Minecraft code working on things to create was way back in 1.10.  Other than that I used other programs to to input the stuff I wanted to try.  Any documentation for the current version, just gave back search results on how villager trades worked, not where to find the file to check what the trades were currently set at, and how to change it.  

Oh and on one note, I think the Amethyst trade as you have it right now, is just fine being as its not easy to get shards of it to start with.

**Reply by BarchamMal** [reply]
By the way, the file changed is called VillagerTrades.java and you can find it in java.barch.mc_extended.Villagers.
